packages/dart/sshnoports/lib/src/print_version.dart

```diff
@@ -1,9 +1,10 @@
 import 'dart:io';
 
-import 'package:sshnoports/src/version.dart' as release;
-import 'package:noports_core/version.dart' as package;
+import 'package:sshnoports/src/version.dart' as binaries;
+import 'package:noports_core/version.dart' as core;
 
 /// Print version number
 void printVersion() {
-  stderr.writeln('Version : ${release.packageVersion} (core: ${package.packageVersion}');
+  stderr.writeln('Version : ${binaries.packageVersion}'
+      ' (core: ${core.packageVersion}');
```

Comment:
Missing the closing bracket for (core: x.x.x)

Reply:
Thanks - will fix in next PR